core/src/proxy/block_engine_stage.rs

```diff
@@ -194,18 +197,48 @@ impl BlockEngineStage {
     ) -> crate::proxy::Result<()> {
         // Get a copy of configs here in case they have changed at runtime
         let keypair = cluster_info.keypair().clone();
+        let local_config = &block_engine_config.lock().unwrap().clone();
+
+        let mut auth_service_endpoint =
+            Endpoint::from_shared(local_config.block_engine_url.clone()).map_err(|_| {
+                ProxyError::AuthenticationConnectionError(
+                    "invalid block engine url value".parse().unwrap(),
+                )
+            })?;
+        let mut backend_endpoint = Endpoint::from_shared(local_config.block_engine_url.clone())
+            .map_err(|_| {
+                ProxyError::BlockEngineConnectionError(
+                    "invalid block engine url value".parse().unwrap(),
+                )
+            })?
+            .tcp_keepalive(Some(Duration::from_secs(60)));
```

Comment:
did we have this in master? mid-review, will circle back to it

Reply:
it's in master 
https://github.com/jito-foundation/jito-solana/blob/4ff92c7205d73837add2c2c28f1a1d12ebaccc48/validator/src/main.rs#L2689-L2692